rs/nervous_system/integration_tests/tests/sns_lifecycle.rs

```diff
@@ -2,6 +2,7 @@ use crate::sns::root::get_sns_canisters_summary;
 use assert_matches::assert_matches;
 use candid::{Nat, Principal};
 use canister_test::Wasm;
+use futures::{stream, StreamExt};
```

Comment:
IIUC, we are suppose to try to stick to tokio. Ofc, if tokio has no equivalent for these, and as long as they are compatible with tokio, then no problem. Just making sure.

Reply:
The `futures` crate is independent of the choice of executor (e.g. tokio).